staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -196,7 +196,7 @@ func conditionFuncFor(condition string, errOut io.Writer) (ConditionFunc, error)
 		return IsCreated, nil
 
 	case strings.HasPrefix(lowercaseCond, "condition="):
-		conditionName := lowercaseCond[len("condition="):]
+		conditionName := condition[len("condition="):]
```

Comment:
i see an update to the test for jsonpath. do we want to add one for `condition` too?

Reply:
it's fixed for both condition and jsonpath